lib/core/auth/gssapi.js

```diff
@@ -96,6 +52,50 @@ class GSSAPI extends AuthProvider {
 }
 module.exports = GSSAPI;
 
+function getClient(authContext, callback) {
```

Comment:
can we make this a little less ambiguous, maybe `makeKerberosClient`? It's unclear to me which client we are referring to in this context